oresat_configs/edl_cmd_defs.yaml

```diff
@@ -0,0 +1,239 @@
+    response:
+      - name: error_code
+        data_type: uint32
+        description: SDO error code (0 is no error)
+
+  - uid: 0x7
+    name: co_sync
+    description: send a CANopen SYNC message on the CAN bus
+    response:
+      - name: sent
+        data_type: bool
+        description: the CANopen SYNC message was sent successfully
+
+  - uid: 0x8
+    name: opd_sysenable
+    description: enable the OPD subsystem
+    request:
+      - name: enable
+        data_type: bool
+        description: true to enable or false to disable
+        enums:
+          disable: 0
+          enable: 1
+    response:
+      - name: status
+        data_type: bool
+        description: OPD subsystem enable status
+        enums:
+          disabled: 0
+          enabled: 1
+
+  - uid: 0x9
+    name: opd_scan
+    description: scan for all nodes on the OPD
+    response:
+      - name: nodes_found
+        data_type: uint8
+        description: the number of nodes found
+
+  - uid: 0xA
+    name: opd_probe
+    description: probe for a node on the OPD
+    request:
+      - name: opd_addr
+        data_type: uint8
+        description: the id of the OPD node to probe for
+    response:
+      - name: found
+        data_type: bool
+        description: true if the node was found or false if not
+        enums:
+          not_found: 0
+          found: 1
+
+  - uid: 0xB
+    name: opd_enable
+    description: enable / disable a node on the OPD
+    request:
+      - name: opd_addr
+        data_type: uint8
+        description: the id of the OPD node to enable / disable
+      - name: enable
+        data_type: bool
+        description: true to enable or false to disable
+        enums:
+          disable: 0
+          enable: 1
+    response:
+      - name: status
+        data_type: uint8
+        description: OPD node status
+
+  - uid: 0xC
+    name: opd_reset
+    description: reset a node on the OPD
+    request:
+      - name: opd_addr
+        data_type: uint8
+        description: The id of the OPD node to reset
+    response:
+      - name: status
+        data_type: uint8
+        description: OPD node status
+
+  - uid: 0xD
+    name: opd_status
+    description: get the status of a node on the OPD
+    request:
+      - name: opd_addr
+        data_type: uint8
+        description: the id of the OPD node to get the status of
+    response:
+     - name: status
+       data_type: uint8
+       description: OPD node status
+
+  - uid: 0xE
+    name: rtc_set_time
+    description: set the RTC time
+    request:
+      - name: time
+        data_type: uint32
+        description: the Unix time to set the rtc to
+        unit: s
+    response:
+      - name: set
+        data_type: bool
+        description: the RTC time was set successfully
+
+  - uid: 0xF
+    name: time_sync
+    description: |
+      c3 will send OreSat's time sync TPDO over the CAN bus (all nodes that are powered on and care
+      about time will sync to it)
```

Comment:
Did you know that there are [up to 63 different ways](https://stackoverflow.com/a/21699210) of writing a multiline string in YAML? I didn't either. 

Up above in `EdlCommandDefinitions` newlines are being stripped from the descriptions but that can be done right in YAML. Instead of `|` you can use `>-`, that is instead of a literal block you can use a folded block with strip block chomping, and that will remove all newlines from the resulting string.

Reply:
Lol, interesting, replaced the `|` with `>-` in all yamls